2019/intcodeComputer/computer.ts

```diff
@@ -0,0 +1,23 @@
+import { Instruction } from "./enums/index.ts";
+import { parse } from "./parser/index.ts";
+
+function process(instructions: number[]): number[] {
+  const newInstructions = [...instructions];
+  const expressions = parse(instructions);
+  console.log(expressions);
+
+  expressions.forEach((expression) => {
+    if (expression.instruction === Instruction.Add) {
+      const { x, y, outputIndex } = expression.data;
+      newInstructions[outputIndex] = x + y;
```

Comment:
Ignore this for the most part, this is just a test message to see how things work with this thing.